src/components/Inputs/InputCheckbox.js

```diff
@@ -0,0 +1,39 @@
+export const InputCheckbox = props => {
+      const {options, value  } = props
+      const changeFeatureHandler = e => {
+      const value = e.target.value;
+      const isChecked = e.target.checked;
+  
+      if (isChecked) {
+        const newValue = [...props.value, value];
+        props.onChange(newValue);
+      } else {
+        const newValue = props.value.filter(x => x !== value);
+        props.onChange(newValue);

```

Comment:
When you are using some parts of the code in both if and else for example "props.onChange(newValue)" move this outside if else and use it only once

Reply:
It is necessary when now i have only "onChange(newValue)"?